pah_behave/pah_behave/features/data/fixtures.sql

```diff
@@ -1,3 +1,3 @@
 INSERT INTO public.fleet_management_user(
-    password, is_superuser, username, first_name, last_name, email, is_staff, is_active, date_joined)
-    VALUES ('pbkdf2_sha256$120000$B4VUk7IZMZCk$t9nZBMo6q4EghFgeqkTLhstGRTlYCRfHJtGkuNzUTK0=', FALSE, 'solomiiavyshniak@eukr.net', 'Ада', 'Гавриленко', 'solomiiavyshniak@ukr.net', FALSE, TRUE, '2019-02-18');
+    password, is_superuser, username, first_name, last_name, email, is_staff, is_active, date_joined, country)
+    VALUES ('pbkdf2_sha256$120000$B4VUk7IZMZCk$t9nZBMo6q4EghFgeqkTLhstGRTlYCRfHJtGkuNzUTK0=', FALSE, 'solomiiavyshniak@eukr.net', 'Ада', 'Гавриленко', 'solomiiavyshniak@ukr.net', FALSE, TRUE, '2019-02-18', 'EN');
```

Comment:
can we go with `UA` instead? that user has ukrainian name and mail so I'd also stick with ukraine as the country of choice.

Reply:
Changes applied